api/tests/unit/environments/dynamodb/test_unit_dynamo_environment_wrapper.py

```diff
@@ -59,3 +60,18 @@ def test_get_item_raises_object_does_not_exists_if_get_item_does_not_return_any_
     # Then
     with pytest.raises(ObjectDoesNotExist):
         dynamo_environment_wrapper.get_item(api_key)
+
+
+def test_delete(
```

Comment:
Let's be a bit more explicit here. Something like: `test_delete_removes_environment_from_dynamodb_table`.